File: exercicios_usp_programacao/ex1.py
"""
Dada uma seqüência de números inteiros não-nulos, seguida por 0, 
imprimir seus quadrados.
"""

def retornar_quadrados(n):
	return n**2


def main():
	while True:
		valor = int(input("Digite um número: "))
		if valor == 0:
			break
		quadrado = retornar_quadrados(valor)
		print(f"O quadrado de {valor} é igual a {quadrado}")

if __name__ == "__main__":
	main()

File: exercicios_usp_programacao/ex1.c
#include <stdio.h>

int retornar_quadrados(int n){
	return n * n;
}

int main(){
	int valor, quadrado;

	printf("Digite um valor: ");
	scanf("%d", &valor);

	while (valor != 0){
		quadrado = retornar_quadrados(valor);
		printf("%d ao quadrado é igual a %d\n", valor, quadrado);
		printf("Digite um valor: ");
        	scanf("%d", &valor);
	}
}
